.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+---
+title: nightly-ci is not happy
+assignees: JurajSadel
+labels: ci-nightly
+---
```

Comment:
Feels a bit weird to have `.github/ISSUE_TEMPLATE/` and `.github/ISSUE_TEMPLATE.md` to me 🤔

Reply:
True, I haven't noticed :D Will move it into the `.github/ISSUE_TEMPLATE/`